ui/js/controllers/perf/graphs.js

```diff
@@ -1054,7 +1054,8 @@ perf.controller('TestChooserCtrl', function($scope, $uibModalInstance, $http,
                       subtests: $scope.includeSubtests ? 1 : 0 }).then(function(seriesList) {
                           $scope.unselectedTestList = _.sortBy(
                               _.filter(seriesList,
-                                       { platform: $scope.selectedPlatform }), 'name');
+                                       { platform: $scope.selectedPlatform,
+                                         hasSubtests: true }), 'name');
```

Comment:
This is a change to the test chooser controller. We don't want or need to filter the results based on whether they have a subtests property (in fact there are many cases that don't work properly if we do this, such as when we want to include subtests, or a non-subtest doesn't have subtests itself).

If you just take this change out I think we'll be good to land, the rest of the patch looks great.